Salary App/Features/SalaryForm/View/SalaryBreakDownViewController.swift

```diff
@@ -0,0 +1,261 @@
+        salaryMonthlyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryMonthlyLbl
+    }()
+    
+    private lazy var yearlyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryYearlyLbl: UILabel = {
+        let salaryYearlyLbl = UILabel()
+        salaryYearlyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryYearlyLbl
+    }()
+    
+    private lazy var dailyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryDailyLbl: UILabel = {
+        let salaryDailyLbl = UILabel()
+        salaryDailyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryDailyLbl
+    }()
+    
+    private lazy var weeklyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryWeeklyLbl: UILabel = {
+        let salaryDailyLbl = UILabel()
+        salaryDailyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryDailyLbl
+    }()
+    
+    
+    private lazy var hourlyStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var salaryHourlyLbl: UILabel = {
+        let salaryHourlyLbl = UILabel()
+        salaryHourlyLbl.translatesAutoresizingMaskIntoConstraints = false
+        return salaryHourlyLbl
+    }()
+    
+    
+    private lazy var taxStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var taxLbl: UILabel = {
+        let taxLbl = UILabel()
+        taxLbl.translatesAutoresizingMaskIntoConstraints = false
+        return taxLbl
+    }()
+
+    
+    private lazy var niStackView: UIStackView = {
```

Comment:
Can you turn these views i.e NI, Tax etc into a single reusable component. Since they're all similar to you can pass in some data and it will just layout/present it on the screen